src/Log/SentryLogger.php

```diff
@@ -13,9 +13,9 @@
 use SilverStripe\Control\Middleware\TrustedProxyMiddleware;
 use SilverStripe\Core\Injector\Injector;
 use SilverStripe\Dev\Backtrace;
+use SilverStripe\Security\Member;
 use SilverStripe\Security\Security;
 use SilverStripe\Core\Config\Configurable;
-use PhpTek\Sentry\Log\SentryLogger;
```

Comment:
Without first importing `SentryLogger` how is `SentryLogger:factory()` supposed to declare `SentryLogger` as a return type?

Reply:
Classes don't import themselves. It's treated as an error.

Reply:
OK, I assume you've run it though? (Forgive me, I'm doing more Python atm and my PHP is getting rusty)

Reply:
Yeah I'm also doing python more these days too. It's always a hassle going back to PHP (never the other way though haha).